outbound:
  slots_group:
  - {name: "group_A"}
  - {name: "group_B"}
